updating/updating-cluster-prepare.adoc

```diff
@@ -1,25 +1,20 @@
 :_content-type: ASSEMBLY
 [id="updating-cluster-prepare"]
-= Preparing to update to {product-title} 4.12
+= Preparing to update to {product-title} 4.13
 include::_attributes/common-attributes.adoc[]
 :context: updating-cluster-prepare
 
 toc::[]
 
-{product-title} 4.12 uses Kubernetes 1.25, which removed several deprecated APIs.
+{product-title} 4.13 uses Kubernetes 1.26, which removed several deprecated APIs.
 
-A cluster administrator must provide a manual acknowledgment before the cluster can be updated from {product-title} 4.11 to 4.12. This is to help prevent issues after upgrading to {product-title} 4.12, where APIs that have been removed are still in use by workloads, tools, or other components running on or interacting with the cluster. Administrators must evaluate their cluster for any APIs in use that will be removed and migrate the affected components to use the appropriate new API version. After this evaluation and migration is complete, the administrator can provide the acknowledgment.
+A cluster administrator must provide a manual acknowledgment before the cluster can be updated from {product-title} 4.12 to 4.13. This is to help prevent issues after upgrading to {product-title} 4.13, where APIs that have been removed are still in use by workloads, tools, or other components running on or interacting with the cluster. Administrators must evaluate their cluster for any APIs in use that will be removed and migrate the affected components to use the appropriate new API version. After this evaluation and migration is complete, the administrator can provide the acknowledgment.
 
-Before you can update your {product-title} 4.11 cluster to 4.12, you must provide the administrator acknowledgment.
+Before you can update your {product-title} 4.12 cluster to 4.13, you must provide the administrator acknowledgment.
 
 // Removed Kubernetes APIs
 include::modules/update-preparing-list.adoc[leveloffset=+1]
 
-[role="_additional-resources"]
-.Additional resources
-
-* xref:../authentication/understanding-and-managing-pod-security-admission.adoc#understanding-and-managing-pod-security-admission[Understanding and managing pod security admission]
```

Comment:
PodSecurity enforcement was [softened for 4.13](https://issues.redhat.com/browse/OCPBUGS-8710), but we still want folks to make those improvements, even though 4.12-to-4.13 will no longer break unfixed workloads.  So I'm surprised we're dropping this section, but I'm also not an auth subject-matter expert.

Reply:
@wking Dropping this section here because it's no longer related to the Kuberentes APIs that were removed. 

But we're still keeping our warnings about upcoming PSA restricted enforcement planned for 4.14. We're adding this warning into the 4.13 release notes (by https://github.com/openshift/openshift-docs/pull/57448). This is the same note we had in the 4.12 release notes (when we expected it to be enabled in 4.13).